index.html

```diff
@@ -0,0 +1,290 @@
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
```

Comment:
I suggest adding background images just as the figma design has. For this project, part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) was to stick as much as possible to the design
<img width="1433" alt="Screenshot 2022-12-02 at 14 39 40" src="https://user-images.githubusercontent.com/48546992/205285010-84730a18-9b1d-47d3-9c06-639333727785.png">